x-pack/plugins/apm/public/components/app/ServiceMap/icons.ts

```diff
@@ -14,10 +14,12 @@ import {
 import { getAgentIcon } from '../../shared/AgentIcon/get_agent_icon';
 import { getSpanIcon } from '../../shared/span_icon/get_span_icon';
 
-export function iconForNode(node: cytoscape.NodeSingular) {
+export function iconForNode(node: cytoscape.NodeSingular, isDarkMode: boolean) {
   const agentName = node.data(AGENT_NAME);
   const subtype = node.data(SPAN_SUBTYPE);
   const type = node.data(SPAN_TYPE);
 
-  return agentName ? getAgentIcon(agentName) : getSpanIcon(type, subtype);
+  return agentName
+    ? getAgentIcon(agentName, isDarkMode)
```

Comment:
@smith I think we want this to always return `isDarkMode` `false` since we want to always use the light theme icons in the service map. How do I go about this?

Reply:
If you don't care about the active theme you can hardcode it to always use the light theme like:

```suggestion
    ? getAgentIcon(agentName, false)
```

Reply:
@sqren Ah cool, that's what I'm after in this scenario.